lma-meetingassist-setup-stack/README_QBUSINESS.md

```diff
@@ -0,0 +1,28 @@
+# Amazon Q Business integration in LMA
+
+Amazon Q is a new generative AI-powered application that helps users get work done. Amazon Q can become your tailored business expert and let you discover content, brainstorm ideas, or create summaries using your company’s data safely and securely. For more information see: [Introducing Amazon Q, a new generative AI-powered assistant](https://aws.amazon.com/blogs/aws/introducing-amazon-q-a-new-generative-ai-powered-assistant-preview). This feature allows your users to converse with Amazon Q using Live Meeting Assistant to ask questions and get answers based on company data.
+
+## Deploy Amazon Q (your business expert) as a fallback source of answers, using Lambda hooks with CustomNoMatches/no_hits
+
+1. Before proceeding, you will need an existing deployment of a Q Business application. Please reference the AWS docs for creating a new [Q Business application](https://docs.aws.amazon.com/amazonq/latest/qbusiness-ug/create-application.html)
+2. When launching or updating the LMA stack, make the following parameter changes:
+    1. `Meeting Assist Service`: Located under `Meeting Assist Options` Select 'Q_Business (use existing)'
+    2. `AmazonQAppId`: Located under `Meeting Assist Q Business Integration`. Existing Amazon Q Application ID
+    3. `IDCApplicationARN`: ARN of the Identity Center customer managed application created for QBusiness. This will be blank on your first launch of LMA and you will need to update this after initial deployment using `Update Stack`, if you have previously created an LMA stack you may create the application before updating.
```

Comment:
Do you mean to say an update stack after initial QBusiness integration or upgrading to the new stack to use this new feature? Are the instructions to updating the stack provided in a different section? If so, I'd link to it here.

Reply:
You have to create the stack once to get the information you need to create the custom application in Identity Center. You then have to update the LMA stack with the ARN of the application you create. Essentially the workflow for a new LMA deployment will go.

- Run LMA w/ only your AppId filled in.
- Go create the custom application in IDC.
- Update LMA stack with the ARN from the custom application.

I will update the wording to make this more clear.